config/Makefile.mingw

```diff
@@ -101,7 +101,9 @@ SAFE_STRING=false
 BYTECC=$(TOOLPREF)gcc -O -mms-bitfields
 
 ### Additional compile-time options for $(BYTECC).  (For static linking.)
-BYTECCCOMPOPTS=-DCAML_NAME_SPACE -Wall -Wno-unused
+BYTECCCOMPOPTS=\
+  -DCAML_NAME_SPACE -D_WIN32_WINNT=0x0600 \
+  -Wall -Wno-unused -Werror
```

Comment:
I thought we weren't putting -Werror in releases but only for development. Has there been some discussion I haven't seen.